doc-Installing_on_OpenShift_Container_Platform/topics/configuring_external_auth_ocp.adoc

```diff
@@ -0,0 +1,278 @@
+[[configuring-authentication]]
+== Configuring External Authentication to {product-title_short}
+
+After installing {product-title_short}, configure external authentication by updating the `httpd-auth-configs` configuration map on the `httpd` pod to include all necessary configuration files and certificates. 
+
+Upon startup, the `httpd` pod overlays its files with the ones specified in the `auth-configuration.conf` file in the configuration map. This is done by the `initialize-httpd-auth` service that runs before `httpd`.
+
+You can automatically generate an updated configuration map by running the `httpd-configmap-generator` tool in its own pod using the steps in <<automatic-defining-configmap>> (recommended). Alternatively, you can define the configuration map manually using the commands in <<manually-defining-configmap>>.
+
+
+[[automatic-defining-configmap]]
+=== Configuring External Authentication Automatically
+
+HOW TO OBTAIN THE tool.
```

Comment:
@dayleparker Hi Dayle, I believe this was a note you added to self to follow up on how to obtain the  `httpd_configmap_generator` tool. Do you want to remove the text for now and investigate further if required?